classes/quiz_settings.php

```diff
@@ -253,6 +296,12 @@ private function before_save() {
         // Recalculate config and config key.
         $this->compute_config();
         $this->compute_config_key();
+
+        if ($this->get('requiresafeexambrowser') == settings_provider::USE_SEB_UPLOAD_CONFIG) {
+            $this->save_filemanager_sebconfigfile_draftarea();
```

Comment:
Also would be good to cove this with the tests. It could be tricky, but in test we can hack it somehow or refactor code to make it more testable. E.g. move getting draftitemid outside of this class.